tests/acceptance/test_uboot_automation.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/python
+# Copyright 2017 Northern.tech AS
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import copy
+import multiprocessing
+import os
+import re
+import shutil
+import subprocess
+
+# Make sure common is imported after fabric, because we override some functions.
+from common import *
+
+class TestUbootAutomation:
+    def revision_already_checked(self):
+        # Update these two to skip the check. You can obtain them with the same
+        # Git command as used below.
+        expected_poky_rev = "e553de00a372b193c64c267853a84c608d1aac6d"
```

Comment:
Will this mean we will need to update this every time we will update poky? If so can we make some explicit comment saying that?

Reply:
Ok, I will improve the comment.